leetcode/tree.md

```diff
@@ -0,0 +1,15 @@
+# tree
```

Comment:
# Invalid PULL Commits

** Invalid Commit Message: added easier solution **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: Update tree.md **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']